pallets/schemas/src/lib.rs

```diff
@@ -184,16 +188,24 @@ pub mod pallet {
 	pub(super) type CurrentSchemaIdentifierMaximum<T: Config> =
 		StorageValue<_, SchemaId, ValueQuery>;
 
+	/// Storage for message schema info struct data
+	/// - Key: Schema Id
+	/// - Value: [`SchemaInfo`](SchemaInfo)
+	#[pallet::storage]
+	#[pallet::getter(fn get_schema_info)]
+	pub(super) type SchemaInfos<T: Config> =
```

Comment:
Minor and not a blocker. How about `SchemasInfo` since it's not about information about one schema? Or just `SchemaInfo`. I kinda like `SchemaInfo` without the s.

Reply:
ahhh I see. There is a struct called SchemaInfo. dang.